docs/content/3.config.md

```diff
@@ -469,3 +469,9 @@ Rollup specific option.
 ### `commonJS`
 
 Rollup specific option. Specifies additional configuration for the rollup CommonJS plugin.
+
+## Preset options
+
+### `firebaseV2`
```

Comment:
Maybe this should be nested under `firebase` to be consistent with `vercel` @pi0